tests/config/test_omegaconf_config.py

```diff
@@ -671,3 +672,89 @@ def test_custom_resolvers(self, tmp_path):
         assert conf["parameters"]["model_options"]["param1"] == 7
         assert conf["parameters"]["model_options"]["param2"] == 3
         assert conf["parameters"]["model_options"]["param3"] == "my_env_variable"
+
+    def test_globals(self, tmp_path):
```

Comment:
This test is doing quite a lot of things, I'd probably separate it into: 
1. Just checking that the omegaconfigloader has the resolver and you can load the globals with `assert conf["globals"] == globals_config`
2. Check that resolving is done properly (for both params & catalog)
3. Check that nested resolving is done properly

Reply:
Done!